sky/cli.py

```diff
@@ -36,7 +36,7 @@
 import sys
 import textwrap
 import typing
-from typing import Any, Dict, List, Optional, Tuple
+from typing import Any, Dict, List, Optional, Sequence, Tuple
```

Comment:
It seems we are not able to use the `collections.abc.Sequence` for python versions lower than 3.9, according to the mypy [document](https://mypy.readthedocs.io/en/latest/builtin_types.html).
 
It will raise the following error:
```
...
/sky-experiment-dev/sky/skypilot_config.py", line 113, in <module>
    def get_nested(keys: Sequence[str], default_value: Any) -> Any:
TypeError: 'ABCMeta' object is not subscriptable
```